.buildkite/pipelines/pull_request/base.yml

```diff
@@ -20,7 +20,7 @@ steps:
           limit: 3
 
   - command: .buildkite/scripts/steps/build_api_docs.sh
-    label: 'Build API Docs'
+    label: 'Check Types and Build API Docs'
```

Comment:
Combining these is at ~50 min runtime, so almost bumping into the timeout. Do you think it's worth increasing the timeout now, or just keep an eye on it?